appveyor.yml

```diff
@@ -18,6 +18,14 @@ environment:
 
   matrix:
   # onnx-ml
+  - ONNX_ML: 1
+    ONNXPY_DIR: C:\Python35-x64
+    CONDA_PREFIX: C:\Miniconda35-x64
+
+  - ONNX_ML: 1
+    ONNXPY_DIR: C:\Python36-x64
+    CONDA_PREFIX: C:\Miniconda36-x64
```

Comment:
this is not used anymore after this change.

Reply:
conda_prefix is added to PATH below